doc/conf.py

```diff
@@ -1,272 +1,28 @@
-# -*- coding: utf-8 -*-
-#
-# python-blosc documentation build configuration file, created by
-# sphinx-quickstart on Sun May  5 19:38:24 2013.
-#
-# This file is execfile()d with the current directory set to its containing dir.
-#
-# Note that not all possible configuration values are present in this
-# autogenerated file.
-#
-# All configuration values have a default; values that are commented out
-# serve to show the default.
-
-import sys, os
-
-# If extensions (or modules to document with autodoc) are in another directory,
-# add these directories to sys.path here. If the directory is relative to the
-# documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
-
-# Let's do an absolute import of the installed blosc module so that we have a
-# link
-import blosc
-blosc_location = os.path.dirname( blosc.__file__ )
-sys.path.insert(0, os.path.abspath(blosc_location))
-
-# -- General configuration -----------------------------------------------------
-
-# If your documentation needs a minimal Sphinx version, state it here.
-#needs_sphinx = '1.0'
-
-# Add any Sphinx extension module names here, as strings. They can be extensions
-# coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
+project = 'Blosc2'
```

Comment:
The project is called Blosc.  Blosc2 is just the 'second' generation :-)

Reply:
Specially in this case, where Python-Blosc is a wrapper for the first generation.

Reply:
thx